src/main/java/uk/gov/hmcts/reform/jps/model/out/SittingRecordSearchResponse.java

```diff
@@ -15,5 +16,6 @@
 @AllArgsConstructor
 public class SittingRecordSearchResponse {
     private int recordCount;
+    private List<RecordingUser> recordingUsers;
     private List<SittingRecord> sittingRecords;
```

Comment:
Is joHRoles collection required?

{  
```suggestion
    private List<SittingRecord> sittingRecords;
```
  "sittingRecords": [<sittingRecord>],
  "recordsCount": <recordCount>,
  "recordingUsers": [<recordingUser>],
  "joHRoles": ["<johRole>"]
}